.eslintrc-next.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/no-commonjs */
+module.exports = {
+  extends: ['next'],
```

Comment:
Seems like this is not the recommended way, see https://nextjs.org/docs/basic-features/eslint#migrating-existing-config.

Reply:
Thanks!